extension/js/addGoogleSearch.js

```diff
@@ -0,0 +1,24 @@
+
+function addGoogleSearch(){
+    if (window.location.href.includes("viewtopic.php")) {
+        let searchUrl = 'https://www.google.com/search?q={{value}}+site%3Aivelt.com+intitle%3A"{{title}}"'
+        let topicTitle = document.querySelectorAll('.topic-title > a')[0].text;

```

Comment:
I would move down the searchUrl declaration after the topicTtile, and change it to hardcode the title. (and remove the {{value}} placeholder.
`let searchUrl = 'https://www.google.com/search?q=site%3Aivelt.com+intitle%3A"' + topicTitle + '"'`